src/model.coffee

```diff
@@ -82,7 +82,7 @@ class Model
     # Returns null
     @save: (id, data, callback) ->
         @adapter.save id, data, (err, attributes) =>
```

Comment:
I did, but since I reverted a few lines it's not showing up anymore in the diff.